### PR TITLE
Feat/3 murmur api create functions that interact with murmur core

### LINKED ISSUE #3: murmur-api: create functions that interact with `murmur-core`
this functions will be called by the endpoints:
/create
/execute

### PR DESCRIPTION
**Pull request by juangirini**
close #3 

In this PR the functionalities for the `/new` and `/execute` endpoints are completed. These endpoints call the `murmur-lib` and execute the transactions

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,67 +1,142 @@
...
+		for i in 2..request.validity + 2 {
+			// wallet is 'active' in 2 blocks
+			let next_block_number: BlockNumber = current_block_number + i;
+			schedule.push(next_block_number);
+		}
+		// 2. create mmr
+		let (call, mmr_store) = murmur::create(
+			username.to_string(),
+			seed.to_string(),
+			EPHEM_MSK, // TODO: replace with an hkdf?
```
Comment (driemworks) [inline src/main.rs:84]: Just a note: I already created a task to explore this, will likely use: https://github.com/RustCrypto/KDFs/tree/master/hkdf

File: Cargo.toml [hunk 2]
```diff
@@ -10,4 +10,10 @@ license = "MIT-0"
 [dependencies]
 rocket = { version = "0.5.1", features = ["tls", "json"] }
 serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
 bcrypt = "0.10.1"
+murmur = { package = "murmur-lib", git = "https://github.com/ideal-lab5/murmur.git", branch = "feat/3-murmur-api-create-functions-that-interact-with-murmur-core"}
```
Comment (driemworks) [inline Cargo.toml:15]: the latest is merged to main now, can be updated here

**Review by driemworks — APPROVED** [review]
LGTM
but want to note: this is an interim solution while we externalize the chain interaction (tx submission).

**Review by colemanirby — APPROVED** [review]
Looks good to me. I will replace the local storage with mongodb integration after the merge is completed.

**Review by colemanirby — APPROVED** [review]

**Pull request merged**